index.html

```diff
@@ -173,7 +173,45 @@ <h1 class="header">What is Technica?</h1>
             <p>At Technica, we want to challenge our hackers to step out of their comfort zone and try something new, whether that's exploring a new technology, tinkering with hardware, or coding for the very first time! At Technica, we want our hackers to learn, grow, and meet new people, so join us at Technica 2021!</p>
           </div>
           <div class="col-lg-4">
-              <img id="about-image" src="img/hackers-illustration.png" alt="hackers posing in four squares of a polaroid" width="100%" height="100%">
+            <img id="about-image" src="img/hackers-illustration.png" alt="hackers posing in four squares of a polaroid" width="100%" height="100%">
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="what-can-you-do section" id="what-can-you-do">
+      <div class="container">
+        <h1 class="header">What Can You Do At Technica?</h1>
+        <div class="slide"></div>
+        <ul class="nav nav-tabs">
+          <li>
+            <a data-toggle="tab" class="tab active show orange" href="#home">
+              <div class="title orange"><div class="tab active show radio orange" href="#home"></div>
```

Comment:
what are the hrefs for? i see all three use #home, not sure if that's intentional

Reply:
Good catch, forgot to delete from copy/paste! Changed names to be more meaningful too.